### PR TITLE
post create functionality is implemented

### PR DESCRIPTION
**Pull request by tahakalkanli**
Create post functionality is implemented. However it works with static data you can find in `src/fakeAPI.js`.

### CODE REVIEW

File: purbee_frontend/src/components/createpost/post_fields.js [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+                        <LocalizationProvider dateAdapter={AdapterDateFns}>
+                            <DateTimePicker
+                                value={dates[index]}
+                                onChange={(e) => { handleChangeDates(e, index) }}
+                                renderInput={(params) => <TextField variant="outlined" fullWidth {...params} />} />
+                        </LocalizationProvider>
+                    </Grid>
+                ))}
+
+                {postType.fields.locationList.map((dateListField, index) => (
```
Comment (gkberk) [inline purbee_frontend/src/components/createpost/post_fields.js:142]: dateListField seems irrelevant here, is it a typo?

File: purbee_frontend/src/components/createpost/post_fields.js [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+                        xs={12}
+                        sm={12}
+                    >
+                        <Typography
+                            color="textPrimary"
+                            sx={{ mb: 1 }}
+                            variant="subtitle1"
+                            align="left"
+                        >
+                            {dateListField.header}
```
Comment (gkberk) [inline purbee_frontend/src/components/createpost/post_fields.js:154]: also here, did you mean mapListField?

**Review by gkberk — CHANGES_REQUESTED** [review]
Location data is not stored. If you could fix these, it seems awesome! Well done.

**Comment by tahakalkanli** [comment]
required changes fulfilled. backend connection still awaits.

**Pull request merged**